### PR TITLE
Readds liquid

### PR DESCRIPTION
**Pull request by ghost**
Reverts HippieStation/HippieStation#5172

FOR FUCKS SAKE, "IMPROVE DON'T REMOVE" APPLIES HERE

### CODE REVIEW

**Comment by ghost** [comment]
bad implementation, don't force broken stuff on players until it's fixed

i support liquids as a mechanic but as it stands, it needs a lot of work

**Comment by Pyko1** [comment]
would be nice to have dev branches that could be forced for next round.

**Comment by GuyonBroadway** [comment]
thats testmerging and it should be a thing if the server tools are working.

File: hippiestation/code/controllers/subsystem/liquids.dm [hunk 1]
```diff
@@ -1,5 +1,19 @@
 PROCESSING_SUBSYSTEM_DEF(liquids)
 	name = "Liquids"
 	wait = 1
```
Comment (GuyonBroadway) [inline hippiestation/code/controllers/subsystem/liquids.dm]: try setting this to 5 in order to minimise lag so it minimises the times it fires per second.

**Review by KayeArray — APPROVED** [review]
code works

**Review by ghost — CHANGES_REQUESTED** [review]
Code works but needs major changes to not shit up the server

**Comment by ghost** [comment]
I'll rework the code later on

**Pull request closed without merging**